### PR TITLE
new

### PR DESCRIPTION
**Pull request by freekman**

### CODE REVIEW

File: new.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+new
```
Comment (freekman) [inline new.txt:1]: retest me

**Comment by diagramphamola** [comment]
retest me

**Comment by diagramphamola** [comment]
retest me

**Comment by diagramphamola** [comment]
retest me

**Comment by diagramphamola** [comment]
retest me

**Comment by diagramphamola** [comment]
retest me

File: new.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+
```
Comment (diagramphamola) [inline new.txt:2]: retest me

**Comment by diagramphamola** [comment]
retest me

File: new.txt (hunk 2 above)
Comment (diagramphamola) [inline new.txt:2]: retest me

File: new.txt (hunk 1 above)
Comment (diagramphamola) [inline new.txt:1]: retest me

**Pull request merged**